_includes/proposals.html

```diff
@@ -60,7 +60,7 @@ <h4 class="featurelist__item__title flex-grow">
           {% if proposal.presented | size %}
           {% for presentation in proposal.presented %}
             <li class="featurelist__item__presented featurelist__item__tag">
-              <a href="{{ presentation.url }}" title='{{ site.data["proposals"]["last-presented-alt"] }}'>{{ presentation.date }}</a>
+              <a {% if presentation.url %}href="{{ presentation.url }}" {% endif %}title='{{ site.data["proposals"]["last-presented-alt"] }}'>{{ presentation.date }}</a>
```

Comment:
Seems like if there’s no url, it shouldn’t be an a tag at all.

Reply:
It seems to be perfectly valid (the HTML validator never complained). Apparently, the name is &ldquo;placeholder hyperlink&rdquo;.

- https://stackoverflow.com/questions/5292343/is-an-anchor-tag-without-the-href-attribute-safe

Reply:
If we were to change the tag to something else (like span), we would have to add the CSS for it too, which is not ideal.

Reply:
i don’t think anything should look like a link if it’s not going to navigate anywhere, so we’d need css changes regardless.

Reply:
- [ ] [remove href](https://css-tricks.com/how-to-disable-links/#surefire-way-remove-the-href)
- [ ] [disabled](https://css-tricks.com/how-to-disable-links/#thats-not-enough-i-want-something-more-complex-so-that-i-can-look-smarter)
- [ ] [over-engineered](https://css-tricks.com/how-to-disable-links/#bring-in-the-javascript)
- [ ] something else

Someone wrote a blog post! 😂

Reply:
Definitely agree, though. Disabled links should not look identical to non-links. +1 on disabled styled.

Reply:
We could even remove the color from the link.

Reply:
That blog post is actually quite wrong&hellip; Deleted [the link](https://css-tricks.com/how-to-disable-links/#bring-in-the-javascript) and the choices it offered.

Notice that the [`not-allowed` value](https://developer.mozilla.org/es/docs/Web/CSS/cursor#Values) for `cursor` is under the &ldquo;drag & drop&rdquo; section. So, we are for sure _not_ going to do that.

Reply:
I’m not sure it should be too light - the date is the important part, the notes are just extra background info.

Reply:
Thinking about something along the lines of the following (we already have the variable from before).

```scss
a:not([href]) {
    color: $color-dim-gray;
}
```

Refs: https://stackoverflow.com/questions/5128711/styling-a-without-href-attribute

Reply:
I’d still prefer not using an anchor at all if there’s nowhere to navigate - that it’s technically valid isn’t really interesting to me.

Reply:
I will do as you wish (although am not certain on the reasoning).

Reply:
I would find it confusing to have a thing that looks and acts like a link, that has a right-click menu and hover behavior like a link, but that i can't open in a new tab, click to navigate, download the target, or save to bookmarks.

Reply:
anchor without href is semantically weird

Reply:
Here's how it looks visually.

<br>

![image](https://user-images.githubusercontent.com/17770407/103501547-8279c000-4e1c-11eb-8132-6844f880c701.png)